juriscraper/pacer/acms_docket.py

```diff
@@ -32,6 +34,24 @@ def _parse_text(self, text):
         """
         self._acms_json = json.loads(text)
 
+    def query(self, case_id: str):
+        """
+        Queries the court API for case and docket details for a given case ID.
+
+        :param case_id: The unique identifier of the case to query.
+        """
```

Comment:
Can we add the following assertion, as in the appellate docket, to ensure that a session is available?

```
assert self.session is not None, (
            "session attribute of AppellateDocketReport cannot be None."
        )
```

Reply:
Sure!